index.html

```diff
@@ -1,24 +1,234 @@
+            </div>
+            <div class="cover-white">
+                <div class="container">
+                    <div class="top-nav">
+                        <a class="logo" href="/#">
+                            <img src="images/logo.jpeg" alt="logo">
+                        </a>
+                        <div class="humberger" onclick="toggleMobileMenu()">
+                            <div></div>
+                            <div></div>
+                            <div></div>
+                        </div>
+                        <nav>
+                            <ul>
+                                <li class="active">
+                                    <a href="/">Home</a>
+                                </li>
+                                <li>
+                                    <a href="/about.html">About</a>
+                                </li>
+                                <li>
+                                    <a class="nav-link" href="#Program">Program</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#Join">Join</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#Sponsor">Sponsor</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link disabled" href="#News">News</a>
+                                </li>
+                            </ul>
+                        </nav>
+                        <div class="humberger-close"
+                            onclick="toggleMobileMenu()">
+                            <i class="fa fa-times fa-2x" aria-hidden="true"></i>
+                        </div>
+                        <div class=" tag">
+                            <span>
+                                live aware
+                            </span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </header>
+
+        <section>
+            <div class="landing-page">
+                <div class="container">
+                    <div>
+                        <h3 class="greeting">"Hello! Sharing World"</h3>
+                        <h1>live aware global summit 2022</h1>
+                        <p class="about">A joyful celebration believing in the value of openness and sharing, 
+                            creating a positive change with poeple from all over 100 countries is taking place in October 
+                            ,in korea.
+                        </p>
+                        <p class="date">2022.10.17(THUR) ~ 30(FRI)</p>
+                        <p class="location">@ National Museum of Korea, Art Center Nabi and more</p>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
- This section of the page doesn't exactly look like the given design template, please try to fix this 🙏 . Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your Project: 
-- While it is important to personalize the content of this project, it is important you stick to the general design layout (background images, color, margins, padding, font size) as seen in the design template
-- Some parts of this section aren't visible because your navbar is overflowing, please try to fix this issue 🙏 🙏 .
-- Some parts of this section are overflowing into the next section (Main program). Please fix this issue 👍 👍 
<img width="950" alt="pius header" src="https://user-images.githubusercontent.com/73607512/198254482-6d63cf67-ebd1-4e0f-a99f-93b8674d6095.PNG">

Design Template:
<img width="625" alt="header behance" src="https://user-images.githubusercontent.com/73607512/198255257-7d431ad2-be11-4056-b5e9-b8eeab18fe62.PNG">